views/auth/signup.hbs

```diff
@@ -0,0 +1,16 @@
+<form id="form" action="/auth/signup" method="POST">
+
+  <label for="username">Username</label><br />
```

Comment:
Delete this </br> because you can add blank space with CSS space margin or padding

Reply:
temporary form for testing only.